boards/arduino-due/include/periph_conf.h

```diff
@@ -342,6 +343,34 @@ extern "C" {
 #define GPIO_B21_MAP        31
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define MCK_DIV_LB_MAX      (10U)
+#define PWM_NUMOF           (1U)
+#define PWM_0_EN            (1)
+#define PWM_MAX_VALUE       (0xffff)
+//#define PWM_MAX_CHANNELS    4
+
+/* Temporary PC2 -- Pin34 XIO-15 */
+#define PWM_0_PIN           PIO_PC2B_PWML0
+
+/* PWM_0 configuration */
+#define PWM_0_DEV           PWM
+#define PWM_0_PID           ID_PWM
+#define PWM_0_CHANNELS      (4U)
+#define PWM_CHNR_BIAS       (4U)
```

Comment:
_PWM_CHNR_BIAS_ is added to the channel number in _pwm_set()_
Like https://github.com/RIOT-OS/RIOT/pull/3170#discussion-diff-31889242, this also addresses how I handle the channels at the moment. It let "tests/periph_pwm" work with the proposed channels 4-7 as 0-3.